examples/plain-javascript/README.md

```diff
@@ -49,6 +52,9 @@ data?](../../doc/cldr.md).
 
 No action needed here.
 
+**3. Globalize dist files**
+
+If you haven't already done this then [install the development external dependencies](../../README.md#install-development-external-dependencies) and [build the distribution files](../../README.md#build).
```

Comment:
@johnnyreilly, 

There are three possible ways of installing Globalize: (a) by downloading a ZIP or a TAR.GZ, (b) by using a package manager, or (c) by using source files (that doc describes in the [Installation](https://github.com/jquery/globalize/#installation) section).

This extra 3rd requirement isn't actually a requirement if user has followed (a) or (b). It is, though, if user has followed (c).

I know the above sentence has been my own suggestion. But, I confess I didn't like it. I believe this will confuse (a) and (b) users and it could be improved.